### PR TITLE
Change transcriptUploadSlice's reducer to take argument

### PR DESCRIPTION
**Pull request by helias22**
### Change types

- [x] Breaking change (new feature which messes with old one)it
- [x] Adding tests
- [x] Front end change

### Testing
- [x] I have added testing to this PR

### CODE REVIEW

File: front-end/src/store/transcriptUploadSlice.js [hunk 1]
```diff
@@ -6,15 +6,13 @@ export const transcriptUploadSlice = createSlice({
...
   },
   reducers: {
-    setTranscriptUploadStatusTrue: state => {
-      state.isUploaded = true
-    },
-    setTranscriptUploadStatusFalse: state => {
-      state.isUploaded = false
+    setTranscriptUploadStatus: (state, action) => {
+      state.isUploaded = action.payload
+      //action.payload can be any object that is passed to the function
```
Comment (KayleeLChan) [inline front-end/src/store/transcriptUploadSlice.js:11]: I love me some comments

**Review by KayleeLChan — APPROVED** [review]
Everything looks good to me! 💃 💃 💃

File: front-end/src/store/transcriptUploadSlice.js (hunk 1 above)
Comment (WeissSa) [reply front-end/src/store/transcriptUploadSlice.js:11]: Not to be contrarian, but I think this comment could be confusing. action,payload is explicitely a boolean here.

**Review by WeissSa — APPROVED** [review]
Stylish :man_singer:

**Pull request merged**